common/config/rush/pnpm-lock.yaml

```diff
@@ -12390,6 +12390,12 @@ packages:
         optional: true
     resolution:
       integrity: sha512-olV41bKSMm8BdnuMsewT4jqlZ8+3TCARAXjZGT9jcoSnrfUnRCqnMoF9XEeoWjbzObpqF9dRhHQj0Xb9QdF6/w==
+  /jest-react-hooks-shallow/1.5.1:
```

Comment:
Remove. I ended up not using shallow rendering in unittests because they don't work well with React hooks.

The workarounds (and the fragility they add to tests) isn't worth shallow rendering for this simple component.

Reply:
Done